app/javascript/react/screens/App/common/networkKey.js

```diff
@@ -1,4 +1,4 @@
 /**
  * returns a unique network based on current UI model
  */
-export const networkKey = network => `${network.clusterId}\\${network.uid_ems}`;
+export const networkKey = network => `${network.clusterId}\\${network.uid_ems || network.ems_ref}`;
```

Comment:
Don't freak, rhevm is the same, its just osp providers don't have a uid_ems rather they have an ems_ref